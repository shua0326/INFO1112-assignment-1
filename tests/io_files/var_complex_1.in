echo hi
var -s number_text "sort words.txt | grep '[asdfghjkl]' | wc -l"
echo ${number_text}
var -s new_prompt "sort words.txt | grep '[ghjdks]' | sort -r | head -1 | awk {print$6}"
echo ${new_prompt}
ls
