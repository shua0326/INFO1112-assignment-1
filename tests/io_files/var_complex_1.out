>> hi
>> >> 7
>> >> pianoforte
>> io_files
run_tests.sh
words.txt
>> 
